File: src/app/model/dto/ProfessorWorkedHoursDto.ts
export class ProfessorWorkedHoursDto {
  id: number;
  name: string;
  surname: string;
  roleType: string;
  workedHours: number;

  constructor(data: any) {
    this.id = data.id;
    this.name = data.name;
    this.surname = data.surname;
    this.roleType = data.roleType;
    this.workedHours = data.workedHours;
  }
}

File: src/main/java/org/aldo/api/data/dto/ProfessorWorkedHoursDto.java
package org.aldo.api.data.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProfessorWorkedHoursDto {
    private Integer id;
    private String name;
    private String surname;
    private String roleName;
    private Integer workedHours;
}